build.gradle

```
@@ -412,26 +412,22 @@ tasks.register("dockerDistUntar") {
   }
 }
 
-def dockerImageName = "consensys/teku"
-def dockerVariants = [
-  "jdk17",
-  "jdk16",
-]
-def dockerPlatforms = "linux/amd64,linux/arm64"
+def dockerImage = "consensys/dctteku"
```

Need to switch this back 
```suggestion
def dockerImage = "consensys/teku"
```

---

done, cleaned up the pr description too